### ISSUE TITLE
City field is showing Text

### ISSUE THREAD

**Issue by gkarthickuae** [EaDesgin/Magento2-City-Dropdown#91]
Hi, I have installed your extension and it's not showing city as dropdown on checkout page and still showing as text field. Is there any customisation needed to show dropdown? Please let me know.

**Comment by gkarthickuae** [comment]
No one is available to answer for this issue? almost a day gone and no follow up from the coder?!

**Comment by razarehman56** [comment]
yes you are rigt. Have you git the extension for city dropdown if yes then can you please also share it with me too.
